docs/manual/scala/guide/services/code/ServiceClients.scala

```diff
@@ -74,7 +74,7 @@ package circuitbreakers {
           .withCircuitBreaker(CircuitBreaker.identifiedBy("hello2"))
       )
     }
-    //#circuit-breakers
+    //#circuit-breaker
     // @formatter:on
```

Comment:
Nice catch! I wonder why the nightly build validations didn't catch that broken ref... @jroper that's the second smell this week, isn't it?